Tests/ValetIntegrationTests/CloudIntegrationTests.swift

```diff
@@ -40,14 +40,10 @@ class CloudIntegrationTests: XCTestCase
     {
         super.setUp()
         
-        ErrorHandler.customAssertBody = { _, _, _, _ in
-            // Nothing to do here.
-        }
-
-        allPermutations.forEach { testValet in testValet.removeAllObjects() }
+        allPermutations.forEach { testValet in try? testValet.removeAllObjects() }
```

Comment:
Do we care if this fails?  Most importantly, could it cause other tests to fail?

Reply:
fair point that it could cause other tests to fail silently. I’ll make this throw.

Reply:
Resolved in ee56d51a0880476a858e8a71b77f4a37f584f3c4